### PR TITLE
chore(hardware): add a diagnostics testing script for the 96 channel

### PR DESCRIPTION
**Pull request by Laura-Danielle**
Shenzhen needs a script that will allow them to test both the plunger and the pick up motors on the 96 channel. Right now there appears to be some type of firmware bug where the application crashes when 
all three motors are running at the same time so this script separates the motor movements into two separate "tests".

### CODE REVIEW

File: hardware/opentrons_hardware/scripts/high_throughput_tip_handling.py [hunk 1]
```diff
@@ -68,7 +65,6 @@ async def run(args: argparse.Namespace) -> None:
 
     messenger = CanMessenger(driver=driver)
     messenger.start()
-    await messenger.send(node_id=NodeId.broadcast, message=EnableMotorRequest())
```
Comment (fsinapi) [inline hardware/opentrons_hardware/scripts/high_throughput_tip_handling.py:71]: Why is this being removed?

File: hardware/opentrons_hardware/scripts/high_throughput_diagnostics.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    dictConfig(LOG_CONFIG)
+
+    parser = argparse.ArgumentParser(
+        description="96 channel tip handling testing script."
+    )
+    add_can_args(parser)
+    parser.add_argument(
+        "--use_plunger",
+        type=bool,
+        help="Move the plunger motor or gear motors",
```
Comment (fsinapi) [inline hardware/opentrons_hardware/scripts/high_throughput_diagnostics.py]: Definitely a nit but I think the help text is a little vague.

**Review by fsinapi — APPROVED** [review]
LGTM, couple of questions but feel free to merge as-is

File: hardware/opentrons_hardware/scripts/high_throughput_tip_handling.py (hunk 1 above)
Comment (Laura-Danielle) [reply hardware/opentrons_hardware/scripts/high_throughput_tip_handling.py:71]: The move messages will automatically enable the motors, so this isn't needed anymore.

**Review by ahiuchingau — APPROVED** [review]

**Pull request merged**